assets/state-mig-manager/0400_configmap.yaml

```diff
@@ -145,37 +145,75 @@ data:
           mig-devices:
             "1g.12gb": 7
 
+      # H200-141GB
```

Comment:
For readability, can we group all of the H200-141GB configurations together? For example: 
```
# H200-141GB
all-1g.18gb: 
 < snip >
 
all-1g.18gb.me:
  < snip >
.
.

all-7g.141gb:
  < snip > 
```

Reply:
The thing is I've followed the arrangement of the configs that was already in place. 


I do agree that it's better to group the configs by product . Can I do this in a follow up PR?

Reply:
In the follow up MR I can do re-arrange the whole config file

Reply:
Since none of the H200-141GB profiles are applicable to any other SKU (as far as I am aware), I would prefer if we grouped all the H200-141GB configs at the bottom, before the `all-balanced` config, as part of this PR. Then, we can do any additional cleanup in a follow-up. 

The arrangement of the configs gets a bit tricky when some configs / profiles are common between multiple SKUs and some are not.

Reply:
Sure, I will make the change

Reply:
@cdesiniotis Please check now